cf_units/__init__.py

```diff
@@ -863,14 +863,6 @@ def _identity(self):
     def __hash__(self):
         return hash(self._identity())
 
-    def __eq__(self, other):
```

Comment:
My apologies - I think you are right - these are duplicated and not actually used!